tests/parsec-cli-tests.sh

```diff
@@ -101,7 +107,7 @@ test_crypto_provider() {
 test_encryption() {
 # $1 - algorithm
     KEY="anta-key-rsa-encrypt"
-    TEST_STR="$(date) Parsec public key encryption test"
+    TEST_STR="$(date) Parsec public key encryption"
```

Comment:
Why this change?

Reply:
For a RSA 1024 key, the string to be encrypted + its padding should be smaller than the key size.

Reply:
Without this change the script fails.

Reply:
Oh, my bad. Thanks for the clarification!